moviesImporter/scripts/importer.py

```diff
@@ -1,53 +1,49 @@
 import csv
 import sys
 from moviesImporter.models import Movie
-from IPython import embed
+from datetime import datetime
+
 
 def run():
+    startTime = datetime.now()
     csv.field_size_limit(sys.maxsize)
-    with open('moviesImporter/data_to_import/title.basics.tsv') as tsvfile:
-        basics = csv.DictReader(tsvfile, dialect='excel-tab')
-        with open('moviesImporter/data_to_import/title.crew.tsv') as tsvfile:
-            crew_members = list(csv.DictReader(tsvfile, dialect='excel-tab'))
-            
-            for title in basics:
-                if title['titleType'] == 'movie' and title['genres'] and title['startYear']:
-                    found_index = None
-                    for index, crew in enumerate(crew_members):
-                        if crew['tconst'] == title['tconst'] and crew['directors'] != '\\N':
-                            try: 
-                                found_index = index
-                                movie_dict = create_movie_dict(crew, title)
-                                Movie(**movie_dict).save()
-                                break 
-                            except Exception as e:
-                                print(e)
-                                print(f"{title['primaryTitle']} did not save.") 
+    with open('moviesImporter/data_to_import/title.basics.tsv', 'r') as basics_file:
+        basics = csv.DictReader(basics_file, dialect='excel-tab')
+        with open('moviesImporter/data_to_import/title.crew.tsv', 'r') as crew_file:
+            movies_to_save = []
+            crew_file_array = list(csv.reader(crew_file, dialect='excel-tab'))
+            c = 0
+            for index, title in enumerate(basics):
+                if title['titleType'] == 'movie' and title['genres'] and title['genres'] != '\\N' and title['startYear'] and title['startYear'] != '\\N':
+                    crew_line = crew_file_array[index + 1]
+                    if crew_line and crew_line[0] == title['tconst'] and crew_line[1] != '\\N':
+                        try:
+                            movie_dict = create_movie_dict(crew_line, title)
+                            c += 1
+                            movies_to_save.append(Movie(**movie_dict))
+                        except Exception as e:
+                            print(e)
+                            print(f"{title['primaryTitle']} did not save.")
 
-                    if found_index:
-                        del crew_members[found_index]
+            Movie.objects.bulk_create(movies_to_save)
```

Comment:
This was the other major improvement I made.

`bulk_create` lets you save a lot of records at once with just one SQL statement. This is _much_ faster than our first method of saving every record within the loop.